moveit_core/utils/include/moveit/utils/moveit_error_code.h

```diff
@@ -55,7 +55,7 @@ class MoveItErrorCode : public moveit_msgs::msg::MoveItErrorCodes
   {
     val = code.val;
   }
-  explicit operator bool() const
```

Comment:
Why remove this here?

Reply:
We had agreed earlier in the pull request to proceed with caution when it came to the explicit operator on type converters. If this example is safe, I can revert that change if that's what you expected here.